pg_analytics/src/datafusion/context.rs

```diff
@@ -171,23 +172,74 @@ impl ParadeContextProvider {
             options: ConfigOptions::new(),
         })
     }
+
+    fn get_table_source_impl(
+        reference: TableReference,
+    ) -> Result<Arc<dyn TableSource>, ParadeError> {
+        let table_name = reference.table();
+        let schema_name = reference.schema();
+
+        if let Some(schema_name) = schema_name {
+            // If a schema was provided in the query, i.e. SELECT * FROM <schema>.<table>
+            DatafusionContext::with_schema_provider(schema_name, |provider| {
+                let table = task::block_on(provider.table(table_name))
+                    .ok_or(NotFound::Table(table_name.to_string()))?;
+
+                Ok(provider_as_source(table))
+            })
+        } else {
+            // If no schema was provided in the query, i.e. SELECT * FROM <table>
+            // Read all schemas from the Postgres search path and cascade through them
+            // until a table is found
+            let current_schemas = unsafe {
+                direct_function_call::<Array<pg_sys::Datum>>(
+                    pg_sys::current_schemas,
+                    &[Some(pg_sys::Datum::from(true))],
```

Comment:
It seems like this `true` is getting passed through to `current_schemas`, is that correct?

What's the reasoning behind choosing `true` (to include implicit tables)? I don't actually know what the right choice is, but it might be worth adding a comment here to explain the reason.